src/hb-aat-layout-trak-table.hh

```diff
@@ -61,15 +66,48 @@ struct TrackData
   inline bool sanitize (hb_sanitize_context_t *c) const
   {
     TRACE_SANITIZE (this);
+    /* TODO */
     return_trace (c->check_struct (this));
   }
 
+  inline float get_tracking (const void *base, float ptem) const
+  {
+    /* CoreText points are CSS pixels (96 per inch),
+     * NOT typographic points (72 per inch).
+     *
+     * https://developer.apple.com/library/content/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Explained/Explained.html
+     */
+    float csspx = ptem * 96.f / 72.f;
+    Fixed fixed_size;
+    fixed_size.set_float (csspx);
+
+    // TODO: Figure out when to use others
+    const TrackTableEntry trackTableEntry = trackTable[0];
+
+    unsigned int size_index;
+    for (size_index = 0; size_index < nSizes; ++size_index)
+      if ((base+sizeTable)[size_index] >= fixed_size)
+        break;
+
+    // We don't attempt to extrapolate to larger or smaller values
```

Comment:
This is what Mozilla [does](https://github.com/qdot/gecko-hg/blob/2abdbb027f5a2f8a0c632544e148e763cbb5189f/gfx/thebes/gfxMacPlatformFontList.mm#L595) but not per spec, `If the point size and track number required are not found in the tracking table, interpolation or extrapolation is performed to determine the adjustment to the advance width that is required for the font size and tracking number specified.`

Reply:
Do we need 2d track/font size interpolation eventually?

Reply:
I think we have a bfind() that works for this.

Reply:
> Do we need 2d track/font size interpolation eventually?

Yes. Bilinear interpolation.

Reply:
Thank you very much :)